src/terra-dev-site/doc/application/how-to.2/theme-strategy.doc.mdx

```diff
@@ -0,0 +1,166 @@
+
+```jsx
+import { ThemeContext } from 'terra-application/lib/theme';
+```
+
+Use the theme context to read the active theme value.
+
+```jsx
+import { ThemeContext } from 'terra-application/lib/theme';
+
+const ExampleComponent = () => {
+  // Access the active theme context value.
+  const theme = useContext(ThemeContext);
+};
+```
+
+The active theme name is stored in a `className` key in the context value. To apply the theme, bind the theme className to the component using [classnames](https://github.com/JedWatson/classnames). Some additional documentation for CSS Modules and classNames can be found on the [conventions page](https://engineering.cerner.com/terra-ui/contributing/terra-ui/conventions). Please note that an empty or undefined className indicates the default theme is active or a theme provider is missing. The `className` key maps directly to the anticipated theme name. Using the correct theme className is required.
+
+```jsx
+import React from 'react';
+import classNames from 'classnames/bind';
+import { ThemeContext } from 'terra-application/lib/theme';
+import styles from './ThemedComponent.module.scss';
+
+const cx = classNames.bind(styles);
+
+const ExampleComponent = () => {
+  const theme = React.useContext(ThemeContext);
+
+  return (
+    <div className={cx('example-component', theme.className)}>
+      This is an example component.
+    </div>
+  );
+};
+```
+
+## SCSS
+
+Once the theme className has been bound, the component can be appropriately themed. Start by defining the variables for the CSS properties that are expected to change between themes. SCSS variables are defined using [custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/--*). These values should utilize the SCSS [var()](https://developer.mozilla.org/en-US/docs/Web/CSS/var) functionality. The var() function works by applying a variable if the value has been set. If no value has been set the default value is used.
+
+`example-component/ExampleComponent.module.scss`
+
+```scss
+// The values in this file represent the default theme. The default theme is applied using the default values in the var() function.
+:local {
+  .example-component {
+    background-color: var(--terra-application-example-component-background-color, #fff);
+    color: var(--terra-application-example-component-color, inherit);
+    font-size: var(--terra-application-example-component-font-size, 1rem);
+  }
+}
+```
+
+To add themes, create new files for each theme and include them in the original SCSS file. Theme files define variables that are applied when the theme becomes active. Themes must be locally scoped and must use a supported theme class name.
```

Comment:
Should we mention we recommend adding all theme vars, even if the styles match the default ? i.e. don't partial theme?

Reply:
Added a few notes about theming all of the variables: https://github.com/cerner/terra-application/pull/25/commits/2a93997b6888d4c01d6be725178806bdd92ac386